### ISSUE TITLE
Bulk Sync > when preference is LEAD  and the conversation to be synced is CONTACT > it shows as record not present on SF

### ISSUE THREAD

**Issue by HazelColaco19** [ajency/Sync-WhatsApp-Conversation#125]
**Pre- Requisites:**

Set the preference to LEAD and have some contacts created in salesforce with same number as in your conversations

**Describe Bug with Steps:**

1. Click on bulk sync.
2. Pop up for  " Sharang Amonkar " appears, which says the record is not present on Salesforce. 

**Expected behavior:**

If the conversation record new, the same number is present on SF but the preference differs like in the case above. Let us show him a screen which looks like this and has the message " This record is present on SF as a Contact, Kindly get in touch with your administrator " 

Basically, the user should not be asked to create a lead/ contact of their preference when there is already one present in SF but cannot be seen since the administrator does not allow the preference.
